File: all/zone2021/zone2021_a/zone2021_a.cs
class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    int ans = 0;

    for(int i = 0; i < s.Length - 3; i++)
    {
      if(s[i] == 'Z' && s[i + 1] == 'O' && s[i + 2] == 'N' && s[i + 3] == 'e')
      {
        ans++;
      }
    }

    Console.WriteLine(ans);
  }
}

File: all/zone2021/zone2021_a/zone2021_a.py
s = input()
c = 0
ans = 0
while c < 9:
  if s[c] == "Z" and s[c + 1] == "O" and s[c + 2] == "N" and s[c + 3] == "e": ans += 1
  c += 1
print(ans)